JSTests/stress/propogate-int-use-with-value-to-int32.js

```diff
@@ -0,0 +1,9 @@
+function test(e, n) {
+    return e[n >>> 5] |= 128 << 24 - n % 32;
+}
+noInline(test);
+
+arr = [1.1, 1.1, 1.1, 1.1];
+for (let i = 0; i < 1e3; i++) {
+    test(arr, 10 << 5);
+}
```

Comment:
This the new test case found from the web crash which is due to unhandled `ValueToInt32` node in the 2nd BackwardsPropagationPhase run.